cookbooks/fb_yum_repos/libraries/yum_repos_helpers.rb

```diff
@@ -34,11 +34,15 @@ def self.get_default_gpg_key(_node)
           :default =>
             'file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch',
         },
+        :rocky=> {
+          :default =>
+            'https://dl.rockylinux.org/pub/rocky/9/metadata/RPM-GPG-KEY-Rocky-9',
```

Comment:
There's no local path to the key from an RPM?

Reply:
I can double check! This was me experimenting with using fb_cookbooks and trying to get it to play nice with the Rocky family of Linux images. I would want to spend a lot more time working on `fb_helpers` and troubleshooting before formally opening a PR